octoprint_bettergrblsupport/__init__.py

```diff
@@ -544,6 +546,7 @@ def hook_gcode_sending(self, comm_instance, phase, cmd, cmd_type, gcode, *args,
                 if currentTime > self.timeRef + 500:
                     # self._logger.info("x={} y={} z={} f={} s={}".format(self.grblX, self.grblY, self.grblZ, self.grblSpeed, self.grblPowerLevel))
                     self._plugin_manager.send_plugin_message(self._identifier, dict(type="grbl_state",
+                                                                                    grblVersion=self.grblVersion,
```

Comment:
we don't have to add the grbl version to the active state push loop.   we should be storing the grbl version as a plugin setting during the connection phase and simply reading that setting value when needed, whether it is part of a knockout model or w/e.